### PR TITLE
Added the possibility to create a portable copy with the use of command line parameters

### PR DESCRIPTION
**Pull request by LeonarddeR**
- --create-portable: creates a new portable copy and starts it after creation
- --create-portable-silent: creates a new portable copy and shuts down NVDA

Both parameters depend on the --portable-path parameter, which has to contain the path to the new portable copy.

IN the process, added a new checkbox to the create portable copy dialog which allows for starting the new copy after installation.

### CODE REVIEW

**Comment by jcsteh** [comment]
Can you give a real world use case/scenario for this? Unattended installation is useful for a complete unattended system install or a deployment across multiple machines, but portable copies are usually something a particular user would choose to generate for their use, rather than something you would deploy during a system install or across multiple systems. Thanks.

**Comment by leonardder** [comment]
Sure:
      
      
      1. At "BabbageCom, we deliver USB
        flash drives to our customers containing NVDA along with Nuance
        Vocalizer. However, many customers seem to lack the"skills to
        manually update their USB flash drive to"contain the most recent
        version of NVDA. With the inclusion of this pull request, things
        can get much easier for them and for us if we deliver them a
        simple script which deploys the most recent NVDA version to
        their USB flash drive using these command line parameters.
        
        2. Apart from this, every user will be able to update their
        flash drive portable copy with just two command line parameters
        added to the launcher, rather than following the procedure of
        starting a launcher, navigating to the create portable copy
        option, manually filling in the path, creating the copy,
        navigating to the new portable copy path, closing the current
        copy of NVDA and starting the new one.
        
      
      3. These parameters can also be
        used in the case current #4263 code is polished some more.
        Although there are some conserns about automatically updating
        portable copies, we can use the portable-path parameter to have
        the desired portable directory automatically filled in into the
        create portable directory window, for example.
      
      
        
          
            
              
              
            
            
          
          {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/nvaccess/nvda","title":"nvaccess/nvda","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/nvaccess/nvda"}},"updates":{"snippets":[{"icon":"PERSON","message":"@jcsteh in #6329: Can you give a real world use case/scenario for this? Unattended installation is useful for a complete unattended system install or a deployment across multiple machines, but portable copies are usually something a particular user would choose to generate for their use, rather than something you would deploy during a system install or across multiple systems. Thanks.\n"}],"action":{"name":"View Pull Request","url":"https://github.com/nvaccess/nvda/pull/6329#issuecomment-244502123"}}}

**Review by feerrenrut — APPROVED** [review]

**Comment by leonardder** [comment]
Sorry @feerrenrut, realised that command line parameters should be added to the user guide.

File: user_docs/en/userGuide.t2t [hunk 1]
```diff
@@ -2471,6 +2471,9 @@ Following are the command line options for NVDA:
 | None | --no-sr-flag  | Don't change the global system screen reader flag |
 | None | --install | Installs NVDA (starting the newly installed copy) |
 | None | --install-silent | Silently installs NVDA (does not start the newly installed copy) |
+| None | --create-portable | Creates a portable copy of NVDA (starting the newly created copy) |
+| None | --create-portable-silent | Creates a portable copy of NVDA (does not start the newly installed copy) |
+| None | portable-path=PORTABLEPATH | The path where a portable copy will be created |
```
Comment (feerrenrut) [inline user_docs/en/userGuide.t2t]: Should this argument start with `--` ?
Also could you document the dependencies between the arguments?

**Review by feerrenrut — CHANGES_REQUESTED** [review]

**Review by feerrenrut — APPROVED** [review]

**Pull request merged**